rust/datafusion/src/sql/planner.rs

```diff
@@ -343,7 +343,7 @@ impl<'a, S: SchemaProvider> SqlToRel<'a, S> {
         match *limit {
             Some(ref limit_expr) => {
                 let n = match self.sql_to_rex(&limit_expr, &input.schema())? {
-                    Expr::Literal(ScalarValue::Int64(n)) => Ok(n as usize),
+                    Expr::Literal(ScalarValue::Int64(n)) => Ok(n.unwrap() as usize),
```

Comment:
Could we math on `Some(n)` or add `if n.is_some()` on the match arm to be sure we aren't calling `unwrap` on ` None`?